tests/ListView/BaseTest.php

```diff
@@ -76,12 +80,14 @@ public function testItemViewAsString(): void
         Assert::equalsWithoutLE(
             <<<HTML
             <div>
-            <div>
+            <ul>
+            <li>
             <div>Id: 1</div><div>Name: John</div><div>Age: 20</div>
```

Comment:
I think `<span>` is more suitable here considering `<li>`.

Reply:
We need to separate items somehow. With `div` is break line.